main.c

```diff
@@ -834,6 +887,13 @@ main(int argc, char **argv)
 
 	joystick_init();
 
+// first time through for iOS
+#if __APPLE__ && (TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE)
+    SDL_SetEventFilter(HandleAppEvents, NULL);
+    createIosMessageObserver();
+    SDL_StartTextInput();
```

Comment:
What does this do? Is it a hint to show the keyboard? Can we move this outside the #ifdef, because it would be generally useful for other touch-enabled platforms?

Reply:
👍